packages/components/organisms/f-form/src/components/Form.vue

```diff
@@ -120,6 +207,31 @@ export default {
                 }
             });
         }
+    },
+
+    validations () {
+        const validationProperties = { formFields: {} };
```

Comment:
Would be good to know if it would be possible to refactor this at some point, I'm not finding it very easy to read at the moment

Reply:
That should ideally be outside of the view...

Reply:
I've cut it back a little bit, hopefully it's no clearer?
Also now moved it to a mixin